tests/tfm/tfm_regression_test/README.rst

```diff
@@ -42,6 +42,8 @@ Building and running
 
 .. |test path| replace:: :file:`tests/tfm/tfm_regression_test/`
 
+Tests can be found under :file:`/modules/tee/tf-m/tf-m-tests/tests_reg` in the |NCS| folder structure.
+
```

Comment:
Btw, just noticed, this gives two conflicting sentences under the Building and running section.

Reply:
True, it looks bit funny:

_Tests can be found under /modules/tee/tf-m/tf-m-tests/tests_reg in the nRF Connect SDK folder structure.

This test can be found under tests/tfm/tfm_regression_test/ in the nRF Connect SDK folder structure._

The first one attempts to indicate the location of the actual test files so that you can check what is being tested. And the automatically generated part is the location of the tests application.

Maybe change this to "Test files can be found under..." and move it below build_and_run_test.txt?

Reply:
tfm_psa_test has the same issue. Maybe a similar change in there? @mia-ko

Reply:
Yes, let's go with your suggestion. For both tests :)